src/scrabble.js

```diff
@@ -1,5 +1,35 @@
-function scrabble() {
-  // write code here
+function scrabble(word) {
+  if (!word) return 0;
+  let alphabet = {
+ 
+    'A': 1, 'E': 1, 'I': 1, 'O': 1, 'U': 1, 'L': 1, 'N': 1, 'R': 1, 'S': 1, 'T': 1,
+    'D': 2, 'G': 2,
+    'B': 3, 'C': 3, 'M': 3, 'P': 3,
+    'F': 4, 'H': 4, 'V': 4, 'W': 4, 'Y': 4,
+    'K': 5,
+    'J': 8, 'X': 8,
+    'Q': 10, 'Z': 10
+  };
+
+  
+word = word.toUpperCase();
+let wordScore = 0;
+
+ for (let i = 0; i < word.length; i++) {
+ const letter = word[i];
+
+
+ if (alphabet[letter] !== undefined) {
+  wordScore += alphabet[letter];
+ }
```

Comment:
Nice comparison!
Did you have any problem making it work with null as an input?